packages/ui/src/pages/Home/MultisigActionMenu.tsx

```diff
@@ -39,14 +43,14 @@ const MultisigActionMenu = ({ withSendButton = true }: MultisigActionMenuProps)
     <>
       {withSendButton && (
         <Button
-          variant="primary"
+          variant="secondary"
           aria-label="send"
           onClick={() => setIsSendModalOpen(true)}
         >
-          New transaction
+          New
```

Comment:
I think the "new" is misleading a little since we don't know what we're talking about, is it a new multisig? And even more importantly, we need the 3 dots buttons too, because right now you have no ways to change the names if all you have is a multisig! I'd leave it primary button like for proxies, it's a little misplaced above everything, but as I suggested before, I think we can focus on make the header for solo multisigs next.

This is what we have.
![image](https://github.com/ChainSafe/Multix/assets/33178835/95159515-1676-4605-9a1d-728685b20c5d)

previous ask, keeping it above, with primary is fine since it's the only one here.
![image](https://github.com/ChainSafe/Multix/assets/33178835/241c444e-03f7-435e-b002-259554075d30)